test/test_ops.py

```diff
@@ -192,6 +228,48 @@ def wrapped_fn(*args, **kwargs):
 
             self.assertEqual(result, expected)
 
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_jvp', set({
+        # Apprently these support forward AD, but we get "Trying to use forward AD..."
+        xfail('var_mean'),
+        xfail('std_mean'),
+
+        # RuntimeError: expected scalar type double but found float
```

Comment:
I have some vague idea of what is wrong here: https://github.com/pytorch/functorch/issues/274

Reply:
Edit: nevermind